code/src/components/ListHeader.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import ButtonBack from './ButtonBack';
+
+const ListHeader = ({ onClick }) => {
+  const [showMenu, setShowMenu] = useState(false);
+
+  const onMenuClick = () => {
+    setShowMenu(!showMenu);
+  };
+
+  const categories = [
+    {
+      category: 'Popular',
+      path: 'popular',
+    },
+    {
+      category: 'Now Playing',
+      path: 'now_playing',
```

Comment:
I seems that "now playing" is the default when the app starts. Could you make it visible to the user what list is presented?